tests/end2end/data/userscripts/test_stdinclose.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python3
```

Comment:
Please remove the `test_*` from the filename - otherwise pytest will try to collect tests from it.